### PR TITLE
docs(playground): add remaining basic usage

### PR DESCRIPTION
**Pull request by liamdebeasi**
Adds the following playgrounds:

- Programmatically toggling accordions
- Listening for ionChange
- Showing multiple accordions at a time

### CODE REVIEW

File: static/usage/accordion/toggle/javascript.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      </div>
+    </ion-accordion>
+  </ion-accordion-group>
+  
+  <ion-button onclick="toggleAccordion()">Toggle Second Accordion</ion-button>
+</ion-content>
+
+<script>
+  const accordionGroup = document.querySelector('ion-accordion-group');
+  const toggleAccordion = () => {
```
Comment (amandaejohnston) [inline static/usage/accordion/toggle/javascript.md:35]:
```suggestion
  var toggleAccordion = () => {
```
Currently you get a `toggleAccordion is not defined` error when clicking the button in Stackblitz.

File: static/usage/accordion/multiple/react.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  IonAccordion, 
+  IonAccordionGroup,
+  IonContent,
+  IonItem, 
+  IonLabel
+} from '@ionic/react';
+function Example() {
+  return (
+    <IonContent>
+      <IonAccordionGroup multiple={true} value={['first', 'third']}>
```
Comment (amandaejohnston) [inline static/usage/accordion/multiple/react.md:13]: This is running into this issue: https://github.com/ionic-team/ionic-framework/issues/25041 You may want to set the value via JS instead as a workaround.

**Review by amandaejohnston — CHANGES_REQUESTED** [review]

**Review by amandaejohnston — APPROVED** [review]

**Review by sean-perkins — CHANGES_REQUESTED** [review]

File: static/usage/accordion/listen-changes/angular-ts.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+@Component({
+  selector: 'app-root',
+  templateUrl: 'app.component.html'
+})
+export class AppComponent {  
+  @ViewChild('listenerOut', { static: true }) listenerOut: ElementRef;
+  private values: string[] = ['first', 'second', 'third'];
+
+  accordionGroupChange = (ev: any) => {
```
[the remark this answers is not in the document]
Comment (liamdebeasi) [reply static/usage/accordion/listen-changes/angular-ts.md:12]: There is, but I ran into this issue: https://github.com/ionic-team/ionic-framework/issues/24245

[the remark this answers is not in the document]
Comment (sean-perkins) [reply static/usage/accordion/listen-changes/angular-ts.md:12]: Ok, figured 👍 I ran into the same, debated on having `(ionEvent)="any$(onEvent($event)"` or just type casting any. I think this is good for now. I believe Stencil has merged the upstream fix that was our blocker to getting this bug resolved.

**Review by sean-perkins — APPROVED** [review]

**Pull request merged**